### PR TITLE
"Home work 2"

### PR DESCRIPTION
**Pull request by luka4ok**

### CODE REVIEW

File: home-work-2/style.css [hunk 1]
```diff
@@ -0,0 +1,13 @@
+.parent {
+    width: 500px;
+    height: 500px;
+    background-color: blueviolet;
+    display: flex;
+    justify-content: center;
+    }
```
Comment (vetalcore) [inline home-work-2/style.css]: please fix indentation

**Review by vetalcore — CHANGES_REQUESTED** [review]

File: home-work-2/style.css [hunk 2]
```diff
@@ -1,13 +1,185 @@
...
+    box-sizing: border-box;
+}
+
+/*HEADER*/
+
+#logo {
+    padding-bottom: 40px;
+}
+
+#main-menu li{
```
Comment (vetalcore) [inline home-work-2/style.css]:
```suggestion
#main-menu li {
```

File: home-work-2/style.css [hunk 3]
```diff
@@ -1,13 +1,185 @@
...
+    color: #fff;
+    padding: 0 20px;
+    display: table-cell;
+    vertical-align: middle;
+    height: 100%;
+}
+
+#main-menu a{
```
Comment (vetalcore) [inline home-work-2/style.css]:
```suggestion
#main-menu a {
```

File: home-work-2/style.css [hunk 4]
```diff
@@ -1,13 +1,185 @@
...
+    text-decoration: none;
+    color: inherit;
```
Comment (vetalcore) [inline home-work-2/style.css]: do u really need `inherit` here

File: home-work-2/style.css [hunk 5]
```diff
@@ -1,13 +1,185 @@
...
+
+#sidebar {
+    width: 240px;
+    border-right: 1px solid #c8c8c8;
+    border-left: 1px solid #c8c8c8;
+    float: left;
+}
+
+.sidebar-list li {
+    border-bottom: 1px solid #c8c8c8;
```
Comment (vetalcore) [inline home-work-2/style.css:86]: also u have plugin in VScode to sort lines alphabetically

File: home-work-2/style.css [hunk 6]
```diff
@@ -1,13 +1,185 @@
...
+.main-block {
+    width: 100%;
+    padding: 40px 0px;
+    overflow: auto;
+}
+
+
+/*SIDEBAR*/
+
+#sidebar {
```
Comment (vetalcore) [inline home-work-2/style.css]: i would prefer classes here

File: home-work-2/style.css [hunk 7]
```diff
@@ -1,13 +1,185 @@
...
+    font-size: 14px;
+    font-family: "Open Sans";
+    padding-left: 20px;
+    color: #656363;
+    line-height: 40px;
+}
+
+.sidebar-list a {
+    text-decoration: none;
+    color: inherit;
```
Comment (vetalcore) [inline home-work-2/style.css]: same here for color

File: home-work-2/style.css [hunk 8]
```diff
@@ -1,13 +1,185 @@
...
+    padding-left: 40px;
+    background-color: #f9f9f9;
+    color: #2c2c2c;
+}
+
+/*end of SIDEBAR*/
+
+/*PRODUCT*/
+.product-wraper {
+    width: 678px;
```
Comment (vetalcore) [inline home-work-2/style.css]: let's  discuss that offline

File: home-work-2/style.css [hunk 9]
```diff
@@ -1,13 +1,185 @@
...
+.product-img {
+    border: 1px solid #c8c8c8;
     display: flex;
     justify-content: center;
-    }
+    overflow: hidden;
+}
+
+.product-description {
+    padding-left: 24px;
```
Comment (vetalcore) [inline home-work-2/style.css]: usually u will have some kind of grid with predetermined indentations

File: home-work-2/index.html [hunk 10]
```diff
@@ -1,14 +1,124 @@
...
+    </head>
+    <body class="body">
+        <div class="container">
+            <header class="header">
+                <h1 id="logo">
+                    <a href="">
+                        <img src="img/logo.png" alt="Fitness logo">
+                    </a>
+                </h1>
+                <nav id="main-menu">
```
Comment (vetalcore) [inline home-work-2/index.html]: why id here

File: home-work-2/index.html [hunk 11]
```diff
@@ -1,14 +1,124 @@
...
+                    <ul class="main-menu-list">
```
Comment (vetalcore) [inline home-work-2/index.html:20]: and class here ? :)

File: home-work-2/index.html [hunk 12]
```diff
@@ -1,14 +1,124 @@
...
+                        <li><a href="">Home</a></li>
+                        <li><a href="">Products</a></li>
+                        <li><a href="">About</a></li>
+                        <li><a href="">Contacts</a></li>
+                    </ul>
+                <nav/>
+            </header>
+        <div class="main-block">
+            <div id="sidebar">
```
Comment (vetalcore) [inline home-work-2/index.html]: same here

**Review by vetalcore — CHANGES_REQUESTED** [review]

File: home-work-2/index.html [hunk 13]
```diff
@@ -0,0 +1,124 @@
...
+                    </a>
+                </h1>
+                <nav id="main-menu">
+                    <ul class="main-menu-list">
+                        <li><a href="">Home</a></li>
+                        <li><a href="">Products</a></li>
+                        <li><a href="">About</a></li>
+                        <li><a href="">Contacts</a></li>
+                    </ul>
+                <nav/>
```
Comment (vetalcore) [inline home-work-2/index.html:26]:
```suggestion
                </nav>
```

File: home-work-2/index.html (hunk 10 above)
Comment (vetalcore) [reply home-work-2/index.html]: prefer changing to class

File: home-work-2/style.css [hunk 14]
```diff
@@ -0,0 +1,185 @@
...
+    padding-bottom: 40px;
+}
+
+#main-menu li{
+    color: #fff;
+    padding: 0 20px;
+    display: table-cell;
+    vertical-align: middle;
+    height: 100%;
+}
```
Comment (vetalcore) [inline home-work-2/style.css]: u can add transparent border on this lvl (to have same width on hover)
```suggestion
    border-left: 1px solid transparent;
    border-right: 1px solid transparent;
}
```

File: home-work-2/style.css [hunk 15]
```diff
@@ -0,0 +1,185 @@
...
+
+.sidebar-list li:first-child {
+    border-top: 1px solid #c8c8c8;
+}
+
+.sidebar-list li:hover {
+    padding-left: 40px;
+    background-color: #f9f9f9;
+    color: #2c2c2c;
+}
```
Comment (vetalcore) [inline home-work-2/style.css:111]:
```suggestion
    cursor: pointer;
}
```

File: home-work-2/style.css (hunk 2 above)
Comment (vetalcore) [reply home-work-2/style.css]: or even
```suggestion
.main-menu-list li{
```

File: home-work-2/style.css (hunk 3 above)
Comment (vetalcore) [reply home-work-2/style.css]: or even
```suggestion
.main-menu-list a{
```

File: home-work-2/style.css [hunk 16]
```diff
@@ -0,0 +1,185 @@
...
+    font-family: "Open Sans";
+    font-size: 14px;
+    line-height: 18px;
+    color: #2c2c2c;
+}
+
+/*the end of PRODUCT*/
+
+/*FOOTER*/
+.footer{
```
Comment (vetalcore) [inline home-work-2/style.css]: u can try to play with position: absolute - to have it always stacked to the bottom
(u also need to add appropriate heights to parent containers)

**Review by vetalcore — CHANGES_REQUESTED** [review]

File: home-work-2/style.css (hunk 1 above)
Comment (luka4ok) [reply home-work-2/style.css]: fixed

File: home-work-2/index.html (hunk 13 above)
Comment (luka4ok) [reply home-work-2/index.html:26]: fixed

File: home-work-2/style.css (hunk 2 above)
Comment (luka4ok) [reply home-work-2/style.css]: fixed

File: home-work-2/style.css (hunk 3 above)
Comment (luka4ok) [reply home-work-2/style.css]: fixed

File: home-work-2/style.css (hunk 4 above)
Comment (luka4ok) [reply home-work-2/style.css]: yes, because the default link color is blue.

File: home-work-2/style.css (hunk 5 above)
Comment (luka4ok) [reply home-work-2/style.css:83]: got it

Comment (luka4ok) [reply home-work-2/style.css:86]: fixed

File: home-work-2/style.css (hunk 6 above)
Comment (luka4ok) [reply home-work-2/style.css]: fixed

File: home-work-2/style.css (hunk 7 above)
Comment (luka4ok) [reply home-work-2/style.css]: same as above

File: home-work-2/style.css (hunk 8 above)
Comment (luka4ok) [reply home-work-2/style.css]: ok

File: home-work-2/style.css (hunk 9 above)
Comment (luka4ok) [reply home-work-2/style.css]: that's great

File: home-work-2/index.html (hunk 10 above)
Comment (luka4ok) [reply home-work-2/index.html]: fixed

File: home-work-2/index.html (hunk 11 above)
Comment (luka4ok) [reply home-work-2/index.html:20]: fixed

File: home-work-2/index.html (hunk 12 above)
Comment (luka4ok) [reply home-work-2/index.html]: fixed

File: home-work-2/style.css (hunk 14 above)
Comment (luka4ok) [reply home-work-2/style.css]: fixed

File: home-work-2/style.css (hunk 15 above)
Comment (luka4ok) [reply home-work-2/style.css:111]: fixed

File: exercises/index.html [hunk 17]
```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
```
Comment (vetalcore) [inline exercises/index.html:1]: are this files related to homework 2?

**Review by vetalcore — CHANGES_REQUESTED** [review]

File: home-work-2/style.css [hunk 18]
```diff
@@ -0,0 +1,194 @@
...
+
+/*PRODUCT*/
+.product-wraper {
+    float: right;
+    width: 678px;
+}
+
+.product-img {
+    border: 1px solid #c8c8c8;
+    display: flex;
```
Comment (vetalcore) [inline home-work-2/style.css:123]: u can't use flex in this homework

File: home-work-2/style.css [hunk 19]
```diff
@@ -0,0 +1,194 @@
...
+
+/*FOOTER*/
+.footer {
+    width: 100%;
+    background-color: #e1e1e1;
+    height: 40px;
+    clear: both;
+    position: absolute;
+    bottom: 40px;
+    width: 980px;
```
Comment (vetalcore) [inline home-work-2/style.css:147]: double `width` prop, besides, u wouldn't have gotten this if u have been sorted them alphabetically

File: home-work-2/style.css [hunk 20]
```diff
@@ -0,0 +1,194 @@
...
+    line-height: 40px;
+}
+
+.sidebar-list li a {
+    color: #656363;
+    display: block;
+    height: 100%;
+    padding-left: 20px;
+    text-decoration: none;
+    width: calc(100% - 20px);
```
Comment (vetalcore) [inline home-work-2/style.css:98]: this style doesn't affect a tag anyhow

File: home-work-2/style.css [hunk 21]
```diff
@@ -0,0 +1,194 @@
...
+
+/*the end of PRODUCT*/
+
+/*FOOTER*/
+.footer {
+    width: 100%;
+    background-color: #e1e1e1;
+    height: 40px;
+    clear: both;
+    position: absolute;
```
Comment (vetalcore) [inline home-work-2/style.css:145]: please remove content of product-description block and check if footer is still in the bottom of page

**Review by vetalcore — CHANGES_REQUESTED** [review]

File: home-work-2/style.css (hunk 18 above)
Comment (luka4ok) [reply home-work-2/style.css:123]: fixed

File: home-work-2/style.css (hunk 19 above)
Comment (luka4ok) [reply home-work-2/style.css:147]: fixed

File: home-work-2/style.css (hunk 20 above)
Comment (luka4ok) [reply home-work-2/style.css:98]: fixed
